

In file included from include/linux/dma-mapping.h:18,
                 from include/scsi/scsi_cmnd.h:4,
                 from drivers/usb/storage/scsiglue.c:52:
include/asm/dma-mapping.h: In function `dma_set_mask':
include/asm/dma-mapping.h:126: error: dereferencing pointer to incomplete type
include/asm/dma-mapping.h:128: error: dereferencing pointer to incomplete type

I thought I fixed this last time??

Signed-off-by: Andrew Morton <akpm@osdl.org>
---

 25-akpm/include/asm-x86_64/dma-mapping.h |    2 ++
 1 files changed, 2 insertions(+)

diff -puN include/asm-x86_64/dma-mapping.h~x86_64-merge-2-build-fix include/asm-x86_64/dma-mapping.h
--- 25/include/asm-x86_64/dma-mapping.h~x86_64-merge-2-build-fix	2004-08-07 13:30:35.575072768 -0700
+++ 25-akpm/include/asm-x86_64/dma-mapping.h	2004-08-07 13:30:35.579072160 -0700
@@ -7,6 +7,8 @@
  */
 
 #include <linux/config.h>
+#include <linux/device.h>
+
 #include <asm/scatterlist.h>
 #include <asm/io.h>
 #include <asm/swiotlb.h>
_
